articles/TOC.yml

```diff
@@ -1,5 +1,8 @@
 - name: Welcome
   href: index.md
+- name: Q# Tour
```

Comment:
It looks like the content from #412 got included in this PR as well by accident? Would you be willing to remove it so that it can be reviewed separately? Thanks!

Reply:
Yes, removed